docs/Tutorials/Create-Privacy-enabled-network.md

```diff
@@ -17,335 +17,204 @@ to encrpyt and distribute [private transactions](../Concepts/Privacy/Privacy.md)
 * [Tessera](../HowTo/GetStarted/Install.md#as-release-binaries).
 * [IBFT network as configured in IBFT tutorial](Create-IBFT-Network.md).
```

Comment:
Configured but not running as we will need to run node 0 and 1 with different options (and the port will conflict if they are already running)
Also maybe we could indicate that we only need node0 and 1 not the others.

Reply:
Updated